compiler/ballerina-lang/src/main/java/org/ballerinalang/compiler/CompilerOptionName.java

```diff
@@ -58,6 +58,7 @@ public enum CompilerOptionName {
 
     LOCK_ENABLED("lockEnabled"),
 
+    @Deprecated
```

Comment:
Can we check if this option can be removed?

Reply:
It is used inside `PackageCompilation` and `ManifestBuilder`.
We are not going to remove the implementation we are having for getting the experimental build option from `Ballerina.toml` and passing to compilation right?

Reply:
We dont need it in Ballerina.toml either. The feature is not used anymore.

Reply:
Removed via https://github.com/ballerina-platform/ballerina-lang/pull/35835/commits/6dbeafe0b36c6f6937dade5af90522a48f5ca189